docker/test_env/ethereum_env.go

```diff
@@ -68,28 +70,45 @@ func (b *EthereumNetworkBuilder) WithConsensusType(consensusType ConsensusType)
 	return b
 }
 
-func (b *EthereumNetworkBuilder) WithConsensusLayer(consensusLayer ConsensusLayer) *EthereumNetworkBuilder {
-	b.consensusLayer = &consensusLayer
-	return b
-}
+func (b *EthereumNetworkBuilder) WithDefaultNetworkParticipants(consensusType ConsensusType) *EthereumNetworkBuilder {
+	b.consensusType = &consensusType
+	switch consensusType {
+	case ConsensusType_PoS:
+		consensusLayer := ConsensusLayer_Prysm
+
+		b.participants = []EthereumNetworkParticipant{
+			{
+				ConsensusLayer: consensusLayer,
+				ExecutionLayer: ExecutionLayer_Geth,
+				Count:          1,
+			},
+		}
+	case ConsensusType_PoW:
```

Comment:
Do we want to even bother with this as an option anymore?

Reply:
good question, it's still much faster, when you're sending a lot of trxs. But what we could do here is for example run latest `Erigon` with `--fakepow` flag (I think Besu might support similar mode too).